guides/common/attributes-foreman-deb.adoc

```diff
@@ -1,5 +1,4 @@
 // Attributes only for foreman-deb build
```

Comment:
```suggestion
```

I assume this comment (and the first blank line) can also be removed.

Reply:
I disagree. I think we should have some comments in the attribute files for knowledge-keeping purposes to ease adoption of work for new writers. The source code is messy and difficult to understand when I come to it as a new writer. Keeping comments for new writers is the least we can do.

Reply:
I thought the comment belonged the the removed `:smart-proxy-installation-guide-title:`.  Still, all other "blocks" have comments describing what they're for. If you think the comment holds any value, keep it.